integration_tests/src/main/python/regexp_test.py

```diff
@@ -804,6 +804,58 @@ def test_regexp_replace_unicode_support():
         ),
         conf=_regexp_conf)
 
+@allow_non_gpu('ProjectExec', 'RegExpReplace')
+def test_regexp_replace_fallback():
+    gen = mk_str_gen('[abcdef]{0,2}')
+
+    conf = { 'spark.rapids.sql.regexp.enabled': 'false' }
```

Comment:
nit, here and other places, could use typed constants 
```suggestion
    conf = { 'spark.rapids.sql.regexp.enabled': False }
```